deploy/build.py

```diff
@@ -2,7 +2,7 @@
 import os
 import subprocess
 
-HERE = os.path.dirname(os.path.abspath(__file__))
+HERE = os.getcwd()
```

Comment:
Why did you change this line? I would keep using the existing line and just look for the proper directory if that is the problem.

Reply:
Thanks for asking about this @damianavila!

Before changing this, I got this error when trying to build the docker image:
```
fatal: ambiguous argument 'pilot-hubs/deploy/images/user': unknown revision or path not in the working tree.
```
Which is because de ` was the right call
 build.py` has been moved to the `deploy` dir and the images are still in the old location.

My first thought was to move the images where the build script is (the `deploy` dir), but then I bumped into:
```
File "deploy/build.py", line 36, in last_modified_commit
    return substring_with_alpha(commit_hash)
  File "deploy/build.py", line 26, in substring_with_alpha
    return s[:max(min_len, first_alpha(s))]
  File "deploy/build.py", line 15, in first_alpha
    raise Exception("No alpha characters in string: {}".format(s))
Exception: No alpha characters in string: 
```
Which is from this line here: https://github.com/2i2c-org/pilot-hubs/blob/6105de7428bb9e7442c56fd61aa031ac2e8f5877/deploy/build.py#L59
That builds the new image tag using the last commit that modified the image. And since the images were moved to a new location, then no prev commit that modified them was found.
So I've decided to let the images where they were and tell the `build.py` where to find them. Not sure if this was the best decision though..

Hope I made a bit of sense in explaining my thinking process :D

Reply:
Thanks for the whole explanation!

> So I've decided to let the images where they were and tell the build.py where to find them. Not sure if this was the best decision though..

I think it is a sensible decision, my question was around why you were using `getcwd` to find the images.
If you keep
```python
HERE = os.path.dirname(os.path.abspath(__file__))
```
maybe this works?
```python
tag = last_modified_commit(os.path.join(os.path.dirname(HERE), "images/user")) 
```

NOT tested, btw...

Reply:
> maybe this works?

Yep, it worked! Just added a commit with this. Thank you!

Reply:
Super! I prefer to avoid using `getcwd` in this sort of case (when you are locating files/dirs). Thanks!